modules/openapi-generator/src/main/resources/typescript-fetch/runtime.mustache

```diff
@@ -52,10 +52,13 @@ export class BaseAPI {
         const body = (context.body instanceof FormData || isBlob(context.body))
 	    ? context.body
 	    : JSON.stringify(context.body);
+
+        const headers = Object.assign(context.headers, this.configuration.headers);
```

Comment:
allow overriding configuration headers with custom headers:
```suggestion
        const headers = Object.assign(this.configuration.headers, context.headers);
```

Reply:
My lack of knowledge of typescript! Thanks for the suggestion. Doing the change right now + generating the samples. Should be up in a couple of minutes.